src/debug/ee/debugger.cpp

```diff
@@ -16639,6 +16793,16 @@ HRESULT DebuggerHeap::Init(BOOL fExecutable)
         return HRESULT_FROM_GetLastError();
     }
 #endif
+
+#ifdef FEATURE_PAL
+    m_execMemAllocator = new DebuggerHeapExecutableMemoryAllocator();
```

Comment:
Should probably use the non-throwing new()
`new (std::nothrow) DebuggerHeapExecutableMemoryAllocator();`